design-proposals/vm-memory-snapshot.md

```diff
@@ -0,0 +1,151 @@
+# Overview
+
+Proposal for taking VM RAM snapshot which can be either a stand alone action, or alongside Online VMSnapshot.
```

Comment:
Is `Online VMSnapshot` a known term? The capitalization looks a bit odd to me. Maybe you can add a link to where it is defined.

Reply:
hmm I guess not, it just seemed like to emphasize the action but I guess it is not proper :) ill update it